data/mods/ssb/moves.ts

```diff
@@ -25,8 +25,8 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		// Other useful things
 		noPPBoosts: true, // add this to not boost the PP of a move, not needed for Z moves, dont include it otherwise
 		isZ: "crystalname", // marks a move as a z move, list the crystal name inside
-		zMoveEffect: '', // for status moves, what happens when this is used as a Z move? check data/moves.js for examples
-		zMoveBoost: {atk: 2}, // for status moves, stat boost given when used as a z move
+		zMove: {effect: '', // for status moves}, what happens when this is used as a Z move? check data/moves.js for examples
+		zMove: {boost: {atk: 2}, // for status moves}, stat boost given when used as a z move
```

Comment:
```suggestion
		zMove: {effect: ''}, // for status moves, what happens when this is used as a Z move? check data/moves.js for examples
		zMove: {boost: {atk: 2}}, // for status moves, stat boost given when used as a z move
```

Reply:
Lmao how did this compile? Good ol regex find and replace

Reply:
This entire block is in `/* ... */` which is how it compiled.

Reply:
Gotcha